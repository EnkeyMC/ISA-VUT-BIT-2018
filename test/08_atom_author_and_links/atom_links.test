NAME=Basic Atom link parsing
ARGS=http://www.mocky.io/v2/5bf2a00c2f00007600cfa1bb -u
RC=0
CHECK_OUTPUT=1
