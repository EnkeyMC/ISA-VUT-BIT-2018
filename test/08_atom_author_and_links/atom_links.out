*** xkcd.com ***
I'm a Car
URL: https://xkcd.com/2064/

Carnot Cycle
URL: https://xkcd.com/2063/
